### ISSUE TITLE
having players removed after logoff #156

### ISSUE THREAD

**Issue by thelonehamster** [baxter-oop/AndOre#158]
after longer periods of time, the map gets covered in "afk" people, who just closed the window. a nice feature would be to have all of their buildings unclaimed if they are the last in their corp, and to have their character sprite removed.
 #156

**Comment by JCharante** [comment]
Yeah this is a necessary feature, alongside with logging off we'll set a timer on players that decreases by 1 every server ticks, and if no requests are sent by them to the server by the time the timer hits 0, they'll be removed from the map.
